generate

```diff
@@ -158,7 +158,7 @@ def recurse(aspect, level=0):
                 output += "* `" + subsub_name + " <" + subsub_name + "/README.rst>`_\n"
         else:
             output += "This aspect does not have any sub aspects.\n\n"
-
+        output += "\n"
```

Comment:
@adtac Yeah,but it is showing some syntax error if I run ./generate.
![screenshot from 2017-03-06 17-42-47](https://cloud.githubusercontent.com/assets/21004581/23609803/b97b9c86-0267-11e7-9a78-e6a418e2a5ea.png)